modules/distribution/src/repository/resources/conf/default.json

```diff
@@ -190,5 +190,50 @@
 
 
   "identity.auth_framework.endpoint.tenant_list_enabled" : false,
-  "identity.auth_framework.endpoint.mutual_ssl_manager_enabled" : true
+  "identity.auth_framework.endpoint.mutual_ssl_manager_enabled" : true,
+
```

Comment:
unnecessary new lines

Reply:
These files were moved to the component level and refactored.